movement/movement.c

```diff
@@ -409,14 +415,18 @@ bool app_loop(void) {
         can_sleep = watch_faces[movement_state.current_watch_face].loop(event, &movement_state.settings, watch_face_contexts[movement_state.current_watch_face]);
 
         // Long-pressing MODE brings one back to the first face, provided that the watch face hasn't decided to send them elsewhere
-        // (and we're not currently on the first face).
+        // (and we're not currently on the first face). If we're currently on the first face, a long press
+        // of MODE sends us to the secondary faces (if defined).
         // Note that it's the face's responsibility to provide some way to get to the next face, so if EVENT_MODE_BUTTON_* is
         // used for face functionality EVENT_MODE_LONG_PRESS should probably be handled and next_face() triggered in the face
         // (which would effectively disable the normal 'long press to face 0' behaviour).
         if (event.event_type == EVENT_MODE_LONG_PRESS 
-            && movement_state.current_watch_face > 0 
             && !movement_state.watch_face_changed) {
-            movement_move_to_face(0);
+            if (movement_state.current_watch_face != 0) {
+                movement_move_to_face(0);
+            } else if (MOVEMENT_SECONDARY_FACE_INDEX != MOVEMENT_NUM_FACES) {
```

Comment:
Question: would it make more sense to `#define MOVEMENT_SECONDARY_FACE_INDEX 0` if not defined? This way the conditional could just be `else if (MOVEMENT_SECONDARY_FACE_INDEX)`. There's no scenario where the secondary face will be the first face, and it feels cleaner.